sdk/iot/core/inc/az_iot_core.h

```diff
@@ -58,6 +58,19 @@ typedef enum
   AZ_IOT_STATUS_TIMEOUT = 504,
 } az_iot_status;
 
+/**
+ * @brief Get the #az_iot_status from an int
+ *
+ * @param[in] status_int  The int with the status number.
+ * @param[out]  status  The #az_iot_status* with the status enum.
+ * @return  The #az_result with the result of the get operation
```

Comment:
there are extra spaces here

Reply:
I used tab instead of space. Will change.